### ISSUE TITLE
[ERR_PACKAGE_IMPORT_NOT_DEFINED] on nuxt dev

### ISSUE THREAD

**Issue by BenjaminOddou** [vueuse/schema-org#16]
Hello 👋,

I have a problem when running `nuxt dev` command with the new version `1.0.0-beta.1`

```shell
Nuxt CLI v3.0.0-rc.8                                                                                                                                                              17:01:51
                                                                                                                                                                                  17:01:59
  > Local:    http://localhost:3000/ 

ℹ Using Tailwind CSS from ~/assets/css/tailwind.css                                                                                                              nuxt:tailwindcss 17:01:59

 WARN  [PWA] Workbox is running in development mode                                                                                                                               17:01:59

ℹ Tailwind Viewer: http://localhost:3000/_tailwind/                                                                                                              nuxt:tailwindcss 17:02:00
✔ PWA icons generated in 1068 ms                                                                                                                                                  17:02:01
ℹ Vite server warmed up in 318ms                                                                                                                                                  17:02:03
ℹ Vite client warmed up in 2602ms                                                                                                                                                 17:02:04
✔ Vite server built in 1705ms                                                                                                                                                     17:02:05
✔ Nitro built in 324 ms                                                                                                                                                     nitro 17:02:05
TypeError [ERR_PACKAGE_IMPORT_NOT_DEFINED]: Package import specifier "#vueuse/schema-org/runtime" is not defined in package /Users/User/Documents/nuxt-app/node_modules/@vueuse/schema-org/package.json imported from /Users/User/Documents/nuxt-app/node_modules/@vueuse/schema-org/dist/runtime/composables/useSchemaOrg.mjs
    at new NodeError (node:internal/errors:372:5)
    at throwImportNotDefined (node:internal/modules/esm/resolve:461:9)
    at packageImportsResolve (node:internal/modules/esm/resolve:841:3)
    at moduleResolve (node:internal/modules/esm/resolve:997:21)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_PACKAGE_IMPORT_NOT_DEFINED'
}
```

**Comment by harlan-zw** [comment]
Hey, this should be fixed in [1.0.0-beta.3](https://github.com/vueuse/schema-org/releases/tag/v1.0.0-beta.3)

**Comment by BenjaminOddou** [comment]
Hello @harlan-zw, I have another issue with `"nuxt-schema-org": "^1.0.0-beta.6"`. I am using it along with `"nuxt": "^3.0.0-rc.8"` and when I launch `nuxt dev` command I have :

```shell
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/User/Documents/nuxt-app/node_modules/@vueuse/schema-org/dist/providers/base.mjs' imported from /Users/User/Documents/nuxt-app/node_modules/@vueuse/schema-org/dist/runtime/components/nodes.mjs
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

**Comment by BenjaminOddou** [comment]
Ok solved for me with "nuxt-schema-org": "^1.0.0-beta.8" !
